docs/evaluation/how_to_guides/compare_experiment_results.mdx

```diff
@@ -39,48 +41,38 @@ Click on the regressions or improvements buttons on the top of each column to fi
 
 ## Update baseline experiment
 
-In order to track regressions, you need a baseline experiment against which to compare. This will be automatically assigned as the first experiment in your comparison, but you can
-change it from the dropdown at the top of the page.
-
-![Baseline](./static/select_baseline.png)
+In order to track regressions, you need a baseline experiment against which to compare and a metric to measure.
+The first experiment in your comparison will be used as the baseline by default, but you can change it from the dropdown at the top of the page.
 
-## Select feedback key
+You will also need to select which feedback key (evaluation metric) you want to focus on.
+Again, one will be assigned by default, but you can adjust as needed.
 
-You will also want to select the feedback key (evaluation metric) on which you would like focus on. This can be selected via another dropdown at the top. Again, one will be assigned by
-default, but you can adjust as needed.
+You can configure whether a higher score is better for the selected feedback key. This preference will be stored.
 
-![Feedback](./static/select_feedback.png)
+![Baseline](./static/select_baseline.png)
 
 ## Open a trace
 
 If tracing is enabled for the evaluation run, you can click on the trace icon in the hover state of any experiment cell to open the trace view for that run. This will open up a trace in the side panel.
```

Comment:
```suggestion
If the example you're evaluating is from an ingested [run](/observability/concepts#runs), you can hover over the output cell and click on the trace icon to open the trace view for that run. This will open up a trace in the side panel.
```